prusti-tests/tests/verify/fail/unsupported/static_const.rs

```diff
@@ -0,0 +1,8 @@
+const fn foo() -> usize {
+    1
+}
+
+fn main() { //~ ERROR: [Prusti internal error] tried to encode a projection that accesses the field 0 of a variant without first downcasting its enumeration std::option::Option<std::fmt::Arguments
```

Comment:
This thing is unsupported because of `assert_eq!`, not because of the const, right?

Reply:
Yes, you are right.